GVFS/FastFetch/FastFetchVerb.cs

```diff
@@ -135,6 +135,13 @@ public class FastFetchVerb
             HelpText = "The GUID of the caller - used for telemetry purposes.")]
         public string ParentActivityId { get; set; }
 
+        [Option(
+            "force",
+            Required = false,
+            Default = false,
+            HelpText = "Force FastFetch to download content as if the current repository was just initialized.")]
+        public bool Force { get; set; }

```

Comment:
We should have logic down below that enforces that `--force` (or `--force-checkout`) must be paired with `--checkout`